test/utils/describeConformanceV5.js

```diff
@@ -217,6 +218,7 @@ function testThemeVariants(element, getOptions) {
 }
 
 const fullSuite = {
+  componentProp: testComponentProp,
```

Comment:
👍 I think I removed this in the first iteration with the `Slider`, but then we decided we should keep the prop. We may need to disable this next on some of the converted components tough..

Reply:
It could create a conflict with open PR, yes.